ONSdigital/census-rm-qid-batch-runner
# Change case column names for fpe

# Motivation and Context
For caseRef fpe there's a change to the primary key on the casev2.cases table.
The uac_qid_link table now refers to caze_case_id.
census-rm-qid-batch-runner 

# What has changed
Now refers to caze_case_id , rather than caze_case_ref

# How to test?
You will have install the correct caseprocessor & caseapi branches locally to get the database set up correctly.
Build this image locally.
Run tests, should pass

# Links
https://trello.com/c/71OfRADc/351-spike-performance-test-rabbits-cluster-prod-spec-5-day
